cohort-engine/src/main/java/com/ibm/cohort/engine/CqlEngineWrapper.java

```diff
@@ -71,11 +74,15 @@
  */
 public class CqlEngineWrapper {
 
+	public static final List<String> SUPPORTED_MODELS = Arrays.asList("http://hl7.org/fhir",
```

Comment:
When I created the unit test for USCore, it would not function without mapping the URL in the compiled ELM (which is the USCore  URL) to the data provider. This was added to support that test case.

Reply:
I get why it's there currently and think it should stay as-is for the time being. I was mostly trying to point out that I think it's the intention of the open source folks to always emit ELM that uses the FHIR url rather than the url of a specific IG. Even your test ELM for USCore contains the base FHIR URL (and the unit test seemed to work for me locally if I removed the uscore url from the map above).

I wanted to point out that we may find out that only the `http://hl7.org/fhir` url should be supported (or at least that's all that the cql tooling folks intend to support), and that it's up for the model info/cql tranlator to get that mapping right.